public/app/features/alerting/unified/components/rule-editor/QueryWrapper.tsx

```diff
@@ -180,6 +180,7 @@ export const QueryWrapper = ({
       <div className={styles.wrapper}>
         <QueryEditorRow<AlertDataQuery>
           alerting
+          hideName={!isAdvancedMode}
```

Comment:
nit: prefer `hideRefId` since that's what we're calling it everywhere else in the codebase